packages/react-sdk/src/state/whiteboardManagerImpl.test.ts

```diff
@@ -72,9 +71,24 @@ describe('WhiteboardManagerImpl', () => {
     whiteboardManager.selectActiveWhiteboardInstance(event0, '@user-id');
     expect(destroySpy).not.toHaveBeenCalled();
 
-    whiteboardManager.selectActiveWhiteboardInstance(event1, '@user-id');
+    // Handler for the `Error: Channel not initialized` throw
+    const error = await new Promise<Error | null>((resolve) => {
+      process.once('uncaughtException', (err: Error) => {
+        resolve(err);
```

Comment:
Should this not use `reject` from the Promise API?

Reply:
No. We want to return the error we got from the process.once here so we then can expect this error later on. We cant do an async expect. That would be undefined behaviour in vitest.

This basically says that it waits for the promise forever until the exception happens. When the exception happened the promise resolves and we handle the error using an expect call to ensure its the one we know we get instead of catching an error we did not expect to happen. This ensures that we dont accidentally suppress other exceptions here.